src/rgw/CMakeLists.txt

```diff
@@ -71,7 +71,9 @@ set(librgw_common_srcs
   rgw_sync_module_log.cc
   rgw_sync_module_pubsub.cc
   rgw_pubsub_push.cc
+  rgw_base_pubsub_rest.cc
   rgw_sync_module_pubsub_rest.cc
+  rgw_pubsub_rest.cc
```

Comment:
nit: most of our other rest handler files start with rgw_rest_*

Reply:
the sync module REST implementations filenames end with "_rest"
can change that for this file.